src/App.jsx

```diff
@@ -2,27 +2,28 @@ import Bio from "./components/Bio";
 import { useState } from "react";
 import Navbar from "./components/Navbar";
 import NavModal from "./components/NavModal";
+import Carousel from "./components/Slider/Carousel";
 
 const App = () => {
-
   const [isModalShowing, setIsModalShowing] = useState(false);
-  
+
   const openModal = () => {
     setIsModalShowing(!isModalShowing);
-    document.body.classList.add('overflow-hidden');
-  }
+    document.body.classList.add("overflow-hidden");
+  };
```

Comment:
This is great and definitely works, but lets do it the React way for the challenge. 

Look into how Ref's work in the react documentation and create a ref on the modal and use the Ref.current.style to update these styles.